docs/versioning.rst

```diff
@@ -8,36 +8,35 @@ new major version. The `Conventional Commits
 requests to track the type of changes that have been merged and ensure that the
 version number is increased correctly.
 
-Since ACTS is still under active development not all visible symbols are
-automatically considered part of the public API and as such fall under the
-Semantic Versioning rules. The subset of symbols that are currently part of the
-public API is outlined below.
+Since ACTS is still under active development, so we create **major** versions
+that break API relatively frequently. We try to limit breaking changes to about
+once a month.
 
 Public API
 ----------
 
-At the moment only the following modules of the core library in the ``Core``
-directory contribute to the public API.
+The API surface of ACTS is very large. Everything at the top-level in the ``Acts`` is considered public.
 
-- ``EventData``
-- ``MagneticField``
-- ``Propagator``
-- ``Surfaces``
-- ``Vertexing``
+.. note::
+    If a private a symbol is part of an otherwise public API, this should be
```

Comment:
Maybe we should define what a private symbol is. You refer to `Acts::detail` here, right?